### DISCUSSION TITLE
Connecting existing user account with new phone

### DISCUSSION THREAD

**Discussion by racheldale07** [KL-Psychological-Methodology/ESMira#4] [Ideas]
Hello, a participant has got a new phone in the middle of a study. Can the account they were using on the old phone be connected to an account on a new phone or does they have to get a new User-ID? 
Thanks!

**Comment by JodliDev** [comment]
Hello. Unfortunately there is no official way (yet) to connect different phones with each other. So they would get a new user-ID every time.

There is a way of transferring all data using the developer mode of the app, but I would not suggest guiding participants through that process, since it is meant for tech-savvy people and debugging purposes.
